lib/components/FieldMessage/FieldMessage.tsx

```diff
@@ -35,7 +35,7 @@ export const FieldMessage = ({
     throw new Error(`Invalid tone: ${tone}`);
   }
 
-  if (!message && !reserveMessageSpace) {
+  if (!message && !secondaryMessage && !reserveMessageSpace) {
```

Comment:
This will Ninja release next time. Maybe we should make this explicit as a fix?

Reply:
Yeah agree, i'll pull it out to another PR.